depthai_demo.py

```diff
@@ -13,6 +13,32 @@
 import depthai
 
 print('Using depthai module from: ', depthai.__file__)
+print('Depthai version installed: ', depthai.__version__)
+
+def get_version_from_requirements():
+    with Path('requirements.txt').open() as f:
+        datafile = f.readlines()
+    for line in datafile:
+        if 'depthai' in line:
+            #not commented out
+            if(line.split()[0] != '#'):
```

Comment:
it won't work if comment is done this way
```
#requests
```
I'd make
```suggestion
            if not line.startswith('#'):
```